junit-jupiter-params/src/main/java/org/junit/jupiter/params/provider/CsvFileArgumentsProvider.java

```diff
@@ -53,7 +53,7 @@
 	public void accept(CsvFileSource annotation) {
 		resources = annotation.resources();
 		charset = Charset.forName(annotation.encoding());
-		hasHeaders = annotation.hasHeaders();
+		skipHeaderLines = annotation.skipHeaderLines();
 		settings = new CsvParserSettings();
```

Comment:
There's a `settings.setHeaderExtractionEnabled(true);` option provided by the `CsvParserSettings`... but I like the simple integer-based skip implementation more.